packages/manager/components/settings/apiKey.tsx

```diff
@@ -32,114 +28,81 @@ const StyledText = styled(Text)`
   --pf-c-content--h2--FontWeight: 100;
 `;
 
+const ClipboardBox = styled.div({
+  width: "500px",
+  height: "40px",
+});
```

Comment:
```suggestion
const StyledClipboardBox = styled.div`
  width: 500px,
  height: 40px,
`;
```

Reply:
@soumyadip007 please prefix the name with Styled

Reply:
Done

Reply:
Also use template literals like i've mentioned